### DISCUSSION TITLE
Feature Request: Support LM Studio

### DISCUSSION THREAD

**Discussion by TechMitten** [cline/cline#465] [Feature Requests]
Please add support for local models using LM Studio. Thank you @saoudrizwan

**Comment by Claudioappassionato** [comment]
you use LmStudio. How do you configure the server?

**Reply by TechMitten** [reply]
Click the Start Server button.

**Reply by RaySinner** [reply]
Thanks, it works! ))

**Comment by KertLynx** [comment]
> Please add support for local models using LM Studio. Thank you @saoudrizwan

Yes please :-)
